javascript/javascript1/week1/goodboy.js

```diff
@@ -0,0 +1,16 @@
+var dogYearOfBirth=2017;
+
+var dogYearFuture=2027;
+
+var dogYear=dogYearFuture - dogYearOfBirth;
+
+var shouldShowResultInDogYears;
```

Comment:
This is the conditional you will need to manipulate in order for us to decide whether to use dog years or human years - now we just need to use it .

Reply:
Missed it . Thanks. Changed now.